diff --git a/aarch64-none-linux-gnu/libc/usr/lib64/libc.so b/aarch64-none-linux-gnu/libc/usr/lib64/libc.so
index 036363a..d6e2a45 100644
--- a/aarch64-none-linux-gnu/libc/usr/lib64/libc.so
+++ b/aarch64-none-linux-gnu/libc/usr/lib64/libc.so
@@ -2,4 +2,4 @@
    Use the shared library, but some functions are only in
    the static library, so try that secondarily.  */
 OUTPUT_FORMAT(elf64-littleaarch64)
-GROUP ( /lib64/libc.so.6 /usr/lib64/libc_nonshared.a  AS_NEEDED ( /lib/ld-linux-aarch64.so.1 ) )
+GROUP ( =/lib64/libc.so.6 =/usr/lib64/libc_nonshared.a  AS_NEEDED ( =/lib/ld-linux-aarch64.so.1 ) )
diff --git a/aarch64-none-linux-gnu/libc/usr/lib64/libm.a b/aarch64-none-linux-gnu/libc/usr/lib64/libm.a
index f0586dd..856f0f8 100644
--- a/aarch64-none-linux-gnu/libc/usr/lib64/libm.a
+++ b/aarch64-none-linux-gnu/libc/usr/lib64/libm.a
@@ -1,4 +1,4 @@
 /* GNU ld script
 */
 OUTPUT_FORMAT(elf64-littleaarch64)
-GROUP ( /usr/lib64/libm-2.38.a /usr/lib64/libmvec.a )
+GROUP ( =/usr/lib64/libm-2.38.a =/usr/lib64/libmvec.a )
diff --git a/aarch64-none-linux-gnu/libc/usr/lib64/libm.so b/aarch64-none-linux-gnu/libc/usr/lib64/libm.so
index b88bdf8..6f3014e 100644
--- a/aarch64-none-linux-gnu/libc/usr/lib64/libm.so
+++ b/aarch64-none-linux-gnu/libc/usr/lib64/libm.so
@@ -1,4 +1,4 @@
 /* GNU ld script
 */
 OUTPUT_FORMAT(elf64-littleaarch64)
-GROUP ( /lib64/libm.so.6  AS_NEEDED ( /lib64/libmvec.so.1 ) )
+GROUP ( =/lib64/libm.so.6  AS_NEEDED ( =/lib64/libmvec.so.1 ) )
